engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/text/AnyToDisplayTextNode.java

```diff
@@ -48,7 +47,7 @@ Text convertText(Text self) {
   @CompilerDirectives.TruffleBoundary
   private static Text takePrefix(Text self, final int limit) {
     var prefix = Core_Text_Utils.take_prefix(self.toString(), limit);
-    return Text.create(prefix);
+    return Text.create(prefix + " …");
```

Comment:
When the limit is 80 and we add 4 characters, should we ask for `Core_Text_Utils.take_prefix(self.toString(), limit - 4)` to never exceed the limit?

Reply:
The three dots are a single character but yes should make it limit - 2. Will update.